toqito/matrix_props/is_permutation.py

```diff
@@ -0,0 +1,64 @@
+"""Is matrix a permutation matrix."""
+import numpy as np
+
+def is_permutation(mat: np.ndarray) -> bool:

```

Comment:
You need an extra space between the `import` statement and the function definition.